docs/userGuide/deployingTheSite.md

```diff
@@ -52,18 +52,44 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 ### Deploying to GitHub Pages via Travis CI
 **You can setup [<tooltip content="a platform for Continuous Integration and Delivery (among other things)">Travis CI</tooltip>](https://www.travis-ci.org/) to automatically build and deploy your site on GitHub Pages every time your GitHub repo is updated.**
 
-Here are the steps to set up Travis CI:
 
-1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.com).
+#### Adding your repository to Travis CI
+
+<panel header="{{icon_info}} Travis CI Migration" type="info" expanded>
```

Comment:
Should this be a `box` instead?

Reply:
I made it a panel so that the user can hide it (since they don't need to know about the migration all the time?). Let me know if you prefer it to be a box instead.

Reply:
Sure, can keep as a panel.